lambdatimes/src/components/Carousel/Carousel.js

```diff
@@ -5,29 +5,48 @@ export default class Carousel extends Component {
   constructor(props){
     super(props);
     this.state = {
-
+      carouselItems: [],
+      selectedImageIndex: 0,
     }
+    this.selectedImage = this.selectedImage.bind(this)
   }
   componentDidMount(){
-
+    this.setState({ carouselItems: carouselData })
   }
 
   leftClick = () => {
-
+    this.setState(state => {
+      return {
+        selectedImageIndex: state.selectedImageIndex === 0
+        ? state.carouselItems.length - 1
+        : --state.selectedImageIndex
+      }
+    })
```

Comment:
Nice use of ternary operator for your click methods. Logic is pretty easy to follow and its concise.